tests/test_actions.py

```diff
@@ -1,5 +1,5 @@
 import sys
-from typing import List
+from typing import List, Literal
```

Comment:
This import only works for more recent Python versions (3.8+). If you replace it with:
`from typing_extensions import Literal`
it should work for all Python versions.

Reply:
Thanks for the tip, I don't think I've used typing_extensions since 2019. The test now passes on my local Python 3.7 install.